feature-libs/asm/customer-360/components/customer-360/customer-360.component.html

```diff
@@ -162,24 +162,7 @@ <h3 id="asm-customer-360-title" class="modal-title">
 
     <!-- Modal Body -->
     <div class="cx-dialog-body">
-      <div class="cx-tab-headers" role="tablist" *cxFeatureLevel="'!6.5'">
-        <ng-container *ngFor="let tabHeader of tabs; let i = index">
-          <button
-            #tabHeaderItem
-            class="cx-tab-header"
-            role="tab"
-            [attr.aria-selected]="activeTab === i"
-            attr.aria-controls="asm-customer-360-tab-panel"
-            (click)="selectTab(i)"
-            [class.active]="activeTab === i"
-            (keydown)="switchTab($event, i)"
-            *ngIf="tabHeader.i18nNameKey !== 'customer360.promotionsTab'"
-          >
-            {{ tabHeader.i18nNameKey | cxTranslate }}
-          </button>
-        </ng-container>
-      </div>
-      <div class="cx-tab-headers" role="tablist" *cxFeatureLevel="'6.5'">
```

Comment:
Since Customer360 was never released, we don't need a feature flag.  It was there because one of the tab section(Promotions) was planned for release later but we are releasing it in 6.6.  thanks